pennylane_lightning/src/tests/TestHelpers.hpp

```diff
@@ -110,7 +110,7 @@ isApproxEqual(const std::vector<Data_t, AllocA> &data1,
               const typename Data_t::value_type eps =
                   std::numeric_limits<typename Data_t::value_type>::epsilon() *
                   100) {
-    return data1 == PLApprox(data2);
+    return data1 == PLApprox(data2).epsilon(eps);
```

Comment:
I think it should be as follows,
```suggestion
    return (data1 == PLApprox(data2)).epsilon(eps);
```